packages/react-native-editor/__device-tests__/gutenberg-editor-block-insertion.test.js

```diff
@@ -6,7 +6,7 @@ import { isAndroid, swipeDown, clickMiddleOfElement } from './helpers/utils';
 import testData from './helpers/test-data';
 
 describe( 'Gutenberg Editor tests for Block insertion', () => {
-	it( 'should be able to insert block into post', async () => {
+	it( 'should be able to insert paragraph block with text between blocks on post', async () => {
```

Comment:
Sorry this might seem like a nit, but I originally read the "with text between blocks" as literally having text in-between blocks, which is something that is not valid for Gutenberg so, wdyt about rephrasing to something like:
```suggestion
	it( 'should be able to insert multi-paragraph text, and text to another paragraph block in between', async () => {
```

Reply:
Ah, I see how it can be confusing. I'll update it as suggested. Thanks for the suggestion!